src/lib/pages/library/ui/library-page.svelte

```diff
@@ -22,8 +22,17 @@
 <!-- Favorites -->
 <section class="flex flex-col p-4 gap-4 items-stretch" aria-labelledby="favorites-header">
   <H1 id="favorites-header">Favorites</H1>
-  {#each episodes as episode (episode.id)}
-    <EpisodeCard {episode} />
+
+  {#each liked as id}
```

Comment:
suggestion (non-blocking): use a keyed each block, that way the updates to the likes will be more efficient
```suggestion
  {#each liked as id (id)}
```